lib/api/2.0/nodes.js

```diff
@@ -61,24 +62,45 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value);
+    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
```

Comment:
Reading up more on Cassandra, I guess if we wanted to support it it might make sense to have a wholly different data model given how different Cassandra is from other DBs we might use. At that point though, I guess I'm questioning what value there is in supporting Cassandra altogether?